src/system/Zikula/Module/UsersModule/Resources/views/User/register.tpl

```diff
@@ -1,12 +1,12 @@
 {strip}
     {gt text='New account registration' assign='templatetitle'}
-    {pageaddvar name="javascript" value="system/Zikula/Module/UsersModule/Resources/public/js/Zikula.Users.Common.UserValidation.js"}
+    {pageaddvar name='javascript' value='polyfill'}
+    {pageaddvar name='javascript' value='system/Zikula/Module/UsersModule/Resources/public/js/Zikula.Users.Common.UserValidation.js'}
```

Comment:
Got it. @ZikulaUsersModule/Resources/public/js/Zikula.Users.Common.UserValidation.js